sky/engine/core/css/FontSize.cpp

```diff
@@ -53,7 +53,7 @@ float FontSize::getComputedSizeFromSpecifiedSize(const Document* document, bool
     // However we always allow the page to set an explicit pixel size that is smaller,
```

Comment:
We should probably drop the minimum font size logic entirely, or hoist it up into Dart. The code here has no idea what transforms are being applied, for one.